### PR TITLE
Fixed typos in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Suplanus**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ We use GitHub to manage reviews of pull requests.
 
 * If you are a new contributor, see: [Steps to Contribute](#steps-to-contribute)
 
-* If you have a trivial fix or improvement, go ahead and create a pull request, addressing (with `@...`) a suitable maintainer of this repository (see [CODEOWNERS](CODEOWNERS) of the  repository you want to contribute to) in the description of the pull request.
+* If you have a trivial fix or improvement, go ahead and create a pull request, addressing (with `@...`) a suitable maintainer of this repository (see [CODEOWNERS](CODEOWNERS) of the repository you want to contribute to) in the description of the pull request.
```
Comment (SebastianWolf-SAP) [inline CONTRIBUTING.md:17]: Thanks for that, of course we will accept that!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -14,11 +14,11 @@ We use GitHub to manage reviews of pull requests.
...
 
 * If you plan to do something more involved, please reach out to us and send an [email](mailto:corona-warn-app.opensource@sap.com). This will avoid unnecessary work and surely give you and us a good deal of inspiration.
 
-* Relevant coding style guidelines are available in the respective sub-repositories as they are programming language-dependent.
+* Relevant coding style guidelines are available in the respective sub repositories as they are programming language dependent.
```
Comment (SebastianWolf-SAP) [inline CONTRIBUTING.md:21]: Why do you want to remove the hyphens here? There are several examples on the web with hyphen, especially for language-dependent. I would rather leave it as is.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ The following rule governs documentation contributions:
 
 ## Pull Request Checklist
 
-* Branch from the master branch and, if needed, rebase to the current master branch before submitting your pull request. If it doesn't merge cleanly with master you may be asked to rebase your changes.
+* Branch from the master branch and, if needed, rebase to the current master branch before submitting your pull request. If it doesn't merge cleanly with master, you may be asked to rebase your changes.
```
Comment (SebastianWolf-SAP) [inline CONTRIBUTING.md:49]: Will be accepted, is in line with https://www.grammarly.com/blog/conditional-sentences/

File: CONTRIBUTING.md (hunk 2 above)
Comment (Suplanus) [reply CONTRIBUTING.md:21]: I think hyphen is the german way to connect words, also in english. In my understanding it is not the typical way to describe things in english.

Comment (SebastianWolf-SAP) [reply CONTRIBUTING.md:21]: Don't want to be nitpicking, but as said, there are a lot of examples using the hyphen - including native speakers. So we will keep the hyphen. I'd appreciate if you remove this change from the PR. Thanks in advance!

**Comment by Suplanus** [comment]
Should be fine now.
Thanks for your work 💕🦄

**Comment by SebastianWolf-SAP** [comment]
Thank you very much again! :)

**Pull request merged**